enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -676,42 +677,9 @@ flowchart TD
         ProxyDeployment
         LBService
     end
-    Gateway[Gateway] -.-> ProxyDeployment(Proxy Deployment)
-    Gateway[Gateway] -.-> LBService(LB Service)
     xRoute[xRoute] --> Gateway[Gateway]
 ```
 
-#### Automated and Manual Gateway Deployments
-
-Istio has a feature called [automated deployment](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment)
-that creates an Envoy deployment and service in the same namespace for each Gateway
-if the Gateway's `spec.addresses` field is left unset. It is enabled via the
-`PILOT_ENABLE_GATEWAY_API_DEPLOYMENT_CONTROLLER` env variable.
-
-Conversely, with [manual deployments](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#manual-deployment),
-if the Gateway has the `spec.addresses` field set, then it must manually link
-to an [ingress gateway](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/#configuring-ingress-using-a-gateway).
-The user needs to make their own ingress gateway  service and deployment in the same
-namespace to manually link to, or they need to use the default ingress gateway (if enabled).
-Users can also link a Gateway to a previously created automated deployment belonging to another
-Gateway.
```

Comment:
I would remove the section on manual deployments but keep the other information.